api/contrib/envoy/extensions/filters/network/golang/v3alpha/golang.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.network.golang.v3alpha;
+
+import "google/protobuf/any.proto";
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.network.golang.v3alpha";
+option java_outer_classname = "GolangProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/golang/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: Golang]
```

Comment:
```suggestion
// [#protodoc-title: Golang]

```

i think the ws necessary for the parser to pick up the title (its currently missing)

Reply:
also lets make the title `Golang network filter` otherwise when it appears in search etc its not clear what the link is

Reply:
done, re-title to `Golang network filter`, the same as `Golang http filter`